### DISCUSSION TITLE
HmIP-VDMOT actuators disconnect from adapter

### DISCUSSION THREAD

**Discussion by flexmatics** [nliaudat/esp32_8ch_motor_shield#20] [Q&A] [answered]
Hi,

I realise this may not be directly linked with controller, but for me, when actuator is closing (pressing on) the valve, when it's nearing the closure, few of the actuators pop-off from place (i.e. these plastic clips don't sustain pressure). 

Would it be possible to control the travel of actuator (and limit it somewhat) via settings? Or maybe some other suggestions how to solve this issue?

Thanks.

**Comment by nliaudat** [comment, marked as answer]
You must lower bemf_trigger_X value till the valve do not pop-off
